buf.yaml

```diff
@@ -0,0 +1,11 @@
+version: v2
+modules:
+  - path: src/main/proto
+deps:
+  - buf.build/googleapis/googleapis
+lint:
```

Comment:
If we are going to use the lint, we could add an step on the pipe to validate we are following the basic conventions

Reply:
It is automatically applied as part of the build, so will fail the build if the lint fails

Reply:
nice